assets/fetch-normalized-vms.sql.j2

```diff
@@ -0,0 +1,44 @@
+#standardsql
+WITH
+
+###
+### Apply basic transformations to the Norway VMS positions.
+###
+raw_vms_positions_normalized AS (
+  SELECT
+    message_id as msgid,
+    `world-fishing-827.udfs_v20210701.normalize_shipname`(vessel_name) as shipname,
+    timestamp_utc as timestamp,
+    lat,
+    lon,
+    speed,
+    course,
+    concat(`world-fishing-827.udfs_v20210701.normalize_shipname`(vessel_name), '-', callsign) as ssvid,
```

Comment:
I think a function similar to normalize_shipname exists for calling, can you check that out? make sure it handles null's nicely.

Also, I will only add the `-` in case there is a callsign, if not I will not add it.

Reply:
The function is `world-fishing-827.udfs_v20210701.normalize_callsign` and switches to `null` the empty string and strings with `"none"`, `"null"`, `"nil"` and `"unknown"`.